services/httpd/service.go

```diff
@@ -2,38 +2,64 @@ package httpd
 
 import (
 	"crypto/tls"
+	"expvar"
 	"fmt"
 	"log"
 	"net"
 	"net/http"
 	"os"
 	"strings"
+	"sync"
+
+	"github.com/influxdb/influxdb/monitor"
+)
+
+// statistics gathered by the httpd package.
+const (
+	statRequests           = "requests"
+	statCQServed           = "cq_served"
+	statQueriesServed      = "queries_served"
+	statWriteServed        = "write_served"
+	statPingServed         = "ping_served"
+	statWriteBytesReceived = "write_bytes_rx"
+	statPointsTransmitted  = "points_tx"
```

Comment:
Does this include only count the points if they were successfully written?  Maybe rename to `statsPointsWritten`.  `Transmitted` seems like it's counting the points over the wire but doesn't care whether they were written or failed.  Might be useful to track those to counts separately though.